lib/modules/apostrophe-templates/lib/nunjucks-extensions/component.js

```diff
@@ -0,0 +1,55 @@
+// Implements {{ component 'moduleName:componentName' with dataObject }}
+
+module.exports = function() {
+
+  this.tags = [ 'component' ];
+
+  this.parse = function(parser, nodes, lexer) {
+      // get the tag token
+      const tok = parser.nextToken();
```

Comment:
`tok` seems like sacrificing clarity for terseness.

Reply:
Thanks. Got it from sample code.